### DISCUSSION TITLE
What .NET version is COSMOS using currently?

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2974] [Q&A] [answered]
Is it using 8.0, or is it still using 7.0?

**Comment by retrievaaaa** [comment, marked as answer]
It never used .NET 7, its on .NET 6

Cosmos only supports LTS versions, so .NET 7 support was never added and .NET 8 support is still being worked on

**Reply by winbamstudios** [reply]
k thanks (also didn't it use .net 5 in the past tho)

**Reply by IlifSine** [reply]
k
